extensions/eclipse-che-theia-prefs-provider/jest.config.js

```diff
@@ -0,0 +1,29 @@
+/********************************************************************************
```

Comment:
hello, do we need this file ?
AFAIK until now all jest config was set in package.json files
https://github.com/eclipse/che-theia/blob/466ddd6ec38e3b73e5e6ce93b673ad4c3f316741/plugins/factory-plugin/package.json#L35-L56

Reply:
hello, 
I'll move this config in package.json.